css/dist/toggle-switch-631-rtl.css

```diff
@@ -49,8 +49,9 @@
 	.switch-light label,
 	.switch-light > span,
 	.switch-toggle label,
-	.switch-toggle > span {
-		line-height: 2em;
+	.switch-toggle > span,
+	.switch-light-visual-label {
+		line-height: 2;
```

Comment:
General comment for this CSS part: It is unclear to me why there in many lines there is measurement unit switch from em to pure numbers or pixels.

Reply:
The SCSS compiler generates this, I didn't touch this, so... I don't know.